manifests/0000_26_cloud-controller-manager-operator_11_deployment.yaml

```diff
@@ -40,7 +40,11 @@ spec:
             URL_ONLY_KUBECONFIG=/etc/kubernetes/kubeconfig
           fi
           exec /cluster-controller-manager-operator \
-          --leader-elect \
+          --leader-elect=true \
+          --leader-elect-lease-duration=137s \
+          --leader-elect-renew-deadline=107s \
```

Comment:
CCCMO doesn't have these like leader-elect-renew-deadline and leader-elect-retry-period, these values are taken directly from the code. https://github.com/openshift/cluster-cloud-controller-manager-operator/blob/master/cmd/cloud-config-sync-controller/main.go#L117-L118